go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/vektah/gqlparser/v2 v2.2.0
 	go.uber.org/zap v1.19.1
 	golang.org/x/net v0.0.0-20210614182718-04defd469f4e
+	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
```

Comment:
Given these are just frontend changes, let's un-stage these go.mod and go.sum files and we can loop back on how they got modified in the install process